data.sty

```diff
@@ -0,0 +1,86 @@
+\RequirePackage{glossaries-extra}
+\RequirePackage{titlesec} % lets you change titles
+\RequirePackage{needspace} % tells latex that some things need space below them, or they look funny (e.g. section headers at the end of a page)
+\RequirePackage[margin=3cm]{geometry}
+\RequirePackage[
+pdfstartpage=1,
+hidelinks=true,
+bookmarks=true]%
+{hyperref}
+
+
+% This sets numbering to track all subsubsections, paragraphs, everything.
+\setcounter{secnumdepth}{6}
```

Comment:
This is just to help me remember the incredibly simplistic beauty of this solution.

Reply:
i'm not sure whether this is a good idea, to write the entire document as a list of sections/subsections/etc. the source code does not appear particularly readable either. how do you compare this against your `points`?

Reply:
I'm yet to decide; I feel the decision on how we structure everything is going to have to involve us, latex-y people, sitting down with the lawyer-y people and coming to some understanding. I just want to remind myself when that discussion happens that this is an option.

Reply:
@yjyao - the example file I was given had numbers before every point, so it seems natural to let the engine do this for a number of reasons:

1. Modification would become easy.
2. Any optional subsections would become easy (numbers don't need to be rechecked)
3. Reference becomes easy (with \labe{thing}).

Source-code readability, on the other hand, is important. I think I've been looking at raw latex so long I've lost the common perspective.

Reply:
using enumerate-descendants also supports everything you pointed, plus you don't need to mess with the starred sections. but for this PR i guess it's okay as is.

Reply:
I feel setting the `secnumdepth` is a more pure latex solution, which is nice. Though this should really be brought to #16.

Reply:
moving to #36